features/admin.sms-templates.v1/pages/sms-customization.tsx

```diff
@@ -0,0 +1,457 @@
+                    smsTemplatesList={ availableSmsTemplatesList }
+                    onTemplateSelected={ handleTemplateIdChange }
+                    onLocaleChanged={ handleLocaleChange }
+                />
+
+                <Segment.Group>
+                    <Segment.Group horizontal>
+                        <Segment className="sms-template-segment-content-header">
+                            { t("extensions:develop.smsTemplates.tabs.content.label") }
+                        </Segment>
+                        <Segment>
+                            { t("extensions:develop.smsTemplates.tabs.preview.label") }
+                        </Segment>
+                    </Segment.Group>
+
+                    <Segment.Group horizontal>
+                        <Segment className="sms-template-segment-content">
+                            <SmsCustomizationForm
+                                isSmsTemplatesListLoading={ isSmsTemplatesListLoading || isSmsTemplateLoading }
+                                selectedSmsTemplate={ currentSmsTemplate }
+                                selectedLocale={ selectedLocale }
+                                onTemplateChanged={
+                                    (updatedTemplateAttributes: Partial<SmsTemplate>) =>
+                                        handleTemplateChange(updatedTemplateAttributes) }
+                                onSubmit={ handleSubmit }
+                                onDeleteRequested={ handleDeleteRequest }
+                                readOnly={
+                                    isReadOnly || (isTemplateNotAvailable && !hasSmsTemplateCreatePermissions) }
+                            />
+                        </Segment>
+                        <Segment className="sms-template-segment-preview">
+                            <SmsTemplatePreview
+                                smsTemplate={ selectedSmsTemplate || currentSmsTemplate }
+                            />
+                        </Segment>
+                    </Segment.Group>
+
+                    <Segment padded={ true }>
+                        <Show
+                            when={ featureConfig?.smsTemplates?.scopes?.update }
+                        >
+                            {
+                                (!isTemplateNotAvailable || hasSmsTemplateCreatePermissions) && (
+                                    <SmsCustomizationFooter
+                                        isSaveButtonLoading={ isSmsTemplatesListLoading || isSmsTemplateLoading }
+                                        onSaveButtonClick={ handleSubmit }
+                                    />
+                                )
+                            }
+                        </Show>
+                    </Segment>
+                </Segment.Group>
+
+                <Divider hidden/>
+
+                { smsTemplate && !isSystemTemplate && selectedLocale !== I18nConstants.DEFAULT_FALLBACK_LANGUAGE &&
```

Comment:
Don't use I18nConstants.DEFAULT_FALLBACK_LANGUAGE, use another constant for en_US